kuma/wiki/middleware.py

```diff
@@ -25,68 +20,3 @@ def process_exception(self, request, exception):
             return never_cache(render)(request, '403.html', context,
                                        status=403)
         return None
-
-
-class DocumentZoneMiddleware(MiddlewareBase):
```

Comment:
I think this is the one middleware that we knew made database requests, there are some tests cases that can potentially drop the ``db`` fixture now (future PR)

Reply:
👍 That's a good catch.